### PR TITLE
Adjust BigQuery CI profile

### PR DESCRIPTION
**Pull request by ebyhr**
## Description

TestBigQueryAvroConnectorTest takes about 40 minutes. 
TestBigQueryAvroTypeMapping takes about 10 minutes. 

We shouldn't run those tests in a profile. 

## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

**Comment by ebyhr** [comment]
Still hitting timeout - https://github.com/trinodb/trino/actions/runs/7026873043/job/19120413146?pr=19944

File: plugin/trino-bigquery/pom.xml [hunk 1]
```diff
@@ -595,7 +589,7 @@
 
         <!-- Separate profile for Arrow related tests since they require extra JVM args -->
         <profile>
-            <id>cloud-tests-arrow-and-fte</id>
+            <id>cloud-tests-smoke</id>
```
Comment (hashhar) [inline plugin/trino-bigquery/pom.xml:592]: the name is misleading. It's smoke + "Important" (type-mapping, cleaner, metadata etc.)

**Review by hashhar — COMMENTED** [review]
Idea LGTM. Naming might need improvement. Maybe just `cloud-tests-1` and `cloud-tests-2`.

File: plugin/trino-bigquery/pom.xml (hunk 1 above)
Comment (ebyhr) [reply plugin/trino-bigquery/pom.xml]: Renamed to cloud-tests-1 & cloud-tests-2.

**Pull request merged**